ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot

```diff
@@ -1270,7 +1272,11 @@ msgid "Search Ontario's Data"
 msgstr ""
 
 #: ckanext/ontario_theme/templates/internal/home/snippets/search.html:12
-msgid "Search datasets"
+msgid "Search"
```

Comment:
`msgid "Search"` is already defined in this file on line 1141 (https://github.com/ongov/ckanext-ontario_theme/blob/ckan_2.9_upgrade_intro_section/ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot#L1141) so no need to repeat it here